src/components/TermAccordion.tsx

```diff
@@ -59,14 +60,16 @@ const ExpandIcon = withStyles((theme) => ({
 export interface TermAccordionProps {
   level: number;
   term: TermInfo;
+  connector?: ReactElement;
 }
 
+//TODO: for some reason, if single parent term with no definition is expanded and then redirected to a term with single parent with no definition, the accordion is still open
```

Comment:
It is possible that the component tree remains the same and only the data changes. If that's the case then the accordion component may persist the state. I'd have to look into it deeper to know for sure.